src/dcos_e2e/cluster.py

```diff
@@ -387,13 +381,18 @@ def install_dcos_from_path(
         self,
         build_artifact: Path,
         dcos_config: Dict[str, Any],
+        files_to_copy_to_genconf_dir: Optional[Iterable[Tuple[Path, Path]]
+                                               ] = (),
         log_output_live: bool = False,
     ) -> None:
         """
         Args:
             build_artifact: The `Path` to a build artifact to install DC/OS
                 from.
             dcos_config: The DC/OS configuration to use.
+            files_to_copy_to_genconf_dir: Pairs of host paths to paths on
```

Comment:
This is ignored after `except NotImplementedError`.
Therefore, a user using AWS / Vagrant will see this documentation and it will not work.

Either:
* Document that - that is confusing
* Resolve it in this PR (large PR, but OK)
* Add `files_to_copy_to_genconf_dir` to `Node.install_dcos_from_path` in another PR, and block on that.